core/src/main/java/org/apache/sdap/mudrod/metadata/pre/ApiHarvester.java

```diff
@@ -57,7 +57,8 @@ public Object execute() {
     //remove old metadata from ES
     es.deleteType(props.getProperty(MudrodConstants.ES_INDEX_NAME), props.getProperty(MudrodConstants.RAW_METADATA_TYPE));
     //harvest new metadata using PO.DAAC web services
-    harvestMetadatafromWeb();
+    if(props.getProperty(MudrodConstants.METADATA_DOWNLOAD).equals("1")) 
```

Comment:
I have a feeling that the string should be on the left hand side of the logic as per 
```
if("1".equals(props.getProperty(MudrodConstants.METADATA_DOWNLOAD)))
```

Reply:
Could you explain the difference? The results looks the same to me.

Reply:
https://stackoverflow.com/questions/24656018/string-literal-expressions-should-be-on-the-left-side-of-an-equals-comparison#24658779